internal/storage/sql_provider.go

```diff
@@ -36,50 +43,98 @@ type SQLProvider struct {
 
 	sqlInsertAuthenticationLog     string
 	sqlGetLatestAuthenticationLogs string
+
+	sqlGetExistingTables string
+	sqlCheckTableExists  string
```

Comment:
Maybe I missed something but I don't see where this is actually used. Can't we just simply remove it?